documentation/modules/auxiliary/admin/http/tomcat_ghostcat.md

```diff
@@ -1,31 +1,93 @@
 ## Vulnerable Application
 
-This module can be used to retrieve arbitrary files from anywhere in the web application, including the `WEB-INF` and `META-INF` 
-directories and any other location that can be reached via ServletContext.getResourceAsStream() on Apache Tomcat servers. 
+### Description
+
+This module can be used to retrieve arbitrary files from anywhere in the web application, including the `WEB-INF` and `META-INF`
+directories and any other location that can be reached via ServletContext.getResourceAsStream() on Apache Tomcat servers.
 It also allows the attacker to process any file in the web application as JSP.
 
+### Setup
+
+Running within a docker container:
+
+```
+docker run --name tomcat --rm -p 8080:8080 -p 8009:8009 tomcat:8.5.32
+```
 
 ## Verification Steps
 
-  1. Install the application and start it
-  2. Start msfconsole
-  3. Do: ```auxiliary/admin/http/tomcat_ghostcat```
-  4. Do: ```set rhosts [ip]```
-  5. Do: ```set rport```
-  6. Do: ```set FILENAME```
-  7. Do: ```run```
+1. Install the application and start it
+2. Start msfconsole
+3. Do: `auxiliary/admin/http/tomcat_ghostcat`
+4. Do: `set RHOSTS [ip]`
+5. Do: `set RPORT [port]`
+6. Do: `set FILENAME [filename]`
+7. Do: `run`
 
 ## Options
 
```

Comment:
Shouldn't this options section also be filled in with things like `AJP_PORT` which is a nonstandard option that you have now introduced?

Reply:
It's unlikely an end user would ever change this value since it's a predefined port